gravitee-apim-e2e/ui-test/integration/apim/ui/apis/ui-api-general.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { ADMIN_USER, API_PUBLISHER_USER } from '@fakers/users/users';
+import { ApisFaker } from '@gravitee/fixtures/management/ApisFaker';
+import { PlansFaker } from '@gravitee/fixtures/management/PlansFaker';
+import { ApiEntity, PlanStatus } from '../../../../../lib/management-webclient-sdk/src/lib/models';
+import faker from '@faker-js/faker';
+
+import homePage from "@pageobjects/HomePage/HomePage"
+import apiHome from "@pageobjects/Apis/ApiHome"
+import apiDetails from '@pageobjects/Apis/ApiDetails';
+
+// This is purely for PageObject example and on hold for now until policy studio changes are completed.
+
+
+    describe('API General Page', () => {
+        beforeEach(() => {
+          cy.loginInAPIM(ADMIN_USER.username, ADMIN_USER.password);
+        });
+
+        const HomePage = new homePage()
+        const ApiHome = new apiHome()
+        const ApiDetails = new apiDetails()
+
+        it('Verify Home Page', () => {
+            cy.visit('/')
+            cy.wait(1000)
+            cy.contains('Home board').should('be.visible');
+        });
+        
+        it('Verify General Page Elements', () => {
+            cy.visit('/#!/environments/default/apis/');
+            cy.getByDataTestId('search').click().type("Test")
+            cy.get('input').type('{enter}')
+            cy.wait(1000)
+            cy.getByDataTestId('api_list_edit_button').first().click();
+            // cy.contains('Test (1)').should('be.visible');
+            cy.contains('Design').should('be.visible');
+            cy.contains('General').should('be.visible');
+            cy.contains('Plans').should('be.visible');
+            // ApiDetails.general().click();
+            // ^ I cannot pinpoint the html element for this button.
```

Comment:
Not even sure yet what it is but if it's an element we want to verify that we have to find a solution for it.